doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -250,7 +250,10 @@ Bluetooth Mesh samples
 Bluetooth Fast Pair samples
 ---------------------------
 
-|no_changes_yet_note|
+* Added support for the :zephyr:board:`nrf54lm20dk` board in the following samples:
+
+  * :ref:`fast_pair_locator_tag`
+  * :ref:`fast_pair_input_device`
```

Comment:
```suggestion
* Added support for the :zephyr:board:`nrf54lm20dk` board in all Fast Pair samples.
```

To align with the previous convention:
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-3.0.0.html#bluetooth_fast_pair_samples